cmd/osv-scanner/main.go

```diff
@@ -99,14 +99,25 @@ func run(args []string, stdout, stderr io.Writer) int {
 			},
 			&cli.BoolFlag{
 				Name:  "experimental-call-analysis",
-				Usage: "attempt call analysis on code to detect only active vulnerabilities",
+				Usage: "[Deprecated] attempt call analysis on code to detect only active vulnerabilities",
 				Value: false,
+				Action: func(context *cli.Context, b bool) error {
+					return fmt.Errorf("the experimental-call-analysis flag is no longer supported. Please use the call-analysis and no-call-analysis flags instead")
```

Comment:
nit: I think it would be better to say something like "has been replaced" than "is no longer supported", since we're still _supporting_ call analysis we're just now upgrading it (effectively)